horovod/spark/keras/remote.py

```diff
@@ -301,15 +305,25 @@ def fn(hvd, tf, keras):
         for gpu in gpus:
             tf.config.experimental.set_memory_growth(gpu, True)
         if gpus:
-            tf.config.experimental.set_visible_devices(gpus[hvd.local_rank()], 'GPU')
+            if os.environ.get('CUDA_VISIBLE_DEVICES') is None:
```

Comment:
In the test(non-databricks environment), assert this branch is met.

Reply:
What's the value of CUDA_VISIBLE_DEVICES set in non-databricks environment?